framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -781,6 +783,14 @@ public void run() {
                                     }
                                 });
             }
+            if (hasOption("version")) {
+                String version = getCheckerVersion();
+                if (version == null) {
+                    messager.printMessage(Kind.NOTE, "Version info not available!");
```

Comment:
Under what circumstances is `version` null?  Is that a bug?

Reply:
Its never null for the current version. It can only be null if the tag in release.properties is not found in `XML_FILE`. Should I remove this check altogether and throw an exception if ever version is null?

Reply:
Since that would indicate an unexpected problem, I think that is the best approach.

Reply:
Made the changes accordingly. Please re-review.

Reply:
@PRITI1999 CI does not pass.  Please make it pass before requesting a review.

Reply:
I apologize. CI succeeds now.